### PR TITLE
v9.10 Changelog/Docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by cpoile — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+### Improvements
+
+#### User Interface (UI)
+ - Pre-packaged GitLab plugin version [v1.9.1](https://github.com/mattermost/mattermost-plugin-gitlab/releases/tag/v1.9.1).
+ - Pre-packaged Mattermost Copilot plugin version [v0.8.1](https://github.com/mattermost/mattermost-plugin-ai/releases/tag/v0.8.1).
+ - Pre-packaged Calls plugin version [v0.28.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.28.1).
+ - Re-designed the user profile popover and improved its performance.
+ - Added banner to prompt users to give desktop notification permissions when opening the app.
+ - Increased the width of the profile picture setting to match other user settings.
+ - Improved screen reader support for emoji picker.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:31]:
```suggestion
 - Improved screen reader support for the emoji picker.
```

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Added a metric to track time it takes for the Threads view to load.
+ - Added support for mobile client metrics.
+ - Increased the range of LCP metrics that can be measured.
+ - Added polling of ``getStatusesByIds`` and ``getProfilesByIds`` network calls. The interval of which is configurable with the ``UsersStatusAndProfileFetchingPollIntervalMilliseconds`` configuration variable.
+ - Added defer loading plugins scripts.
+
+### Bug Fixes
+ - Fixed an issue where the ``RefreshPostStats`` job could fail.
+ - Fixed an issue where attempting to create a team with the URL of an existing team showed the wrong error message.
+ - Fixed an issue where ``visibilitychange`` js browser event had not been added for updating the user's current timezone.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:49]: Does ``js`` here mean JavaScript? If so, let's spell it out for clarity.

File: source/deploy/mattermost-changelog.md [hunk 3]
```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue where the last admin could no longer be demoted.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:50]: The last admin in the system, or the last admin added to the system? If it's the last admin in the system, does it make sense to be able to demote that account?

File: source/deploy/mattermost-changelog.md [hunk 4]
```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue where banners set by system administrators did not stack below system banners, and rather appeared underneath them. Existing system banners have remained unchanged.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:51]:
```suggestion
 - Fixed an issue where banners set by system administrators did not stack below system banners, and appeared underneath them instead. Existing system banners have remained unchanged.
```

File: source/deploy/mattermost-changelog.md [hunk 5]
```diff
@@ -11,6 +11,72 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Fixed an issue with an incorrect wrapping of long words in numbered lists.
+ - Fixed an incorrect behavior of image proxy when site URL is changed.
```
Comment (cwarnermm) [inline source/deploy/mattermost-changelog.md:53]:
```suggestion
 - Fixed an incorrect behavior of the image proxy when site URL is changed.
```

**Review by cwarnermm — APPROVED** [review]
Provided editorial feedback inline. Approving for expediency.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Review by agnivade — APPROVED** [review]

**Review by marianunez — APPROVED** [review]

**Review by gabrieljackson — APPROVED** [review]

**Pull request merged**